src/Neo.CLI/CLI/MainService.Plugins.cs

```diff
@@ -90,6 +90,26 @@ private static async Task<Stream> DownloadPluginAsync(string pluginName, Version
                     s["tag_name"]!.GetValue<string>() == $"v{pluginVersion.ToString(3)}" &&
                     s["prerelease"]!.GetValue<bool>() == prerelease) ?? throw new Exception($"Could not find Release {pluginVersion}");
 
+            if (jsonRelease == null)
+            {
+                // If the corresponding version of the plugin is not found, get the latest version
+                jsonRelease = json.AsArray()
+                    .OrderByDescending(s => Version.Parse(s["tag_name"]!.GetValue<string>()?.TrimStart('v')))
```

Comment:
`Version.Parse` will crash if it's `null`. or we have a `tag_name` that isn't valid format.